plotly/graph_reference.py

```diff
@@ -410,12 +410,25 @@ def _patch_objects():
                          'attribute_paths': layout_attribute_paths,
                          'additional_attributes': {}}
 
-    figure_attributes = {'layout': {'role': 'object'},
-                         'data': {'role': 'object', '_isLinkedToArray': True}}
+    figure_attributes = {
+        'layout': {'role': 'object'},
+        'data': {'role': 'object', '_isLinkedToArray': True},
+        'frames': {'role': 'object', '_isLinkedToArray': True}
+    }
     OBJECTS['figure'] = {'meta_paths': [],
                          'attribute_paths': [],
                          'additional_attributes': figure_attributes}
 
+    frame_entry_attributes = {
+        'data': {'role': 'object', '_isLinkedToArray': True},
+        'group': {'role': 'info'},
+        'layout': {'role': 'object'},
+        'name': {'role': 'info'}
+    }
```

Comment:
@etpinard or @rreusser Is this information available anywhere? I don't really want to hard-code these. If it's *not* available, mind helping me to flesh out this list of valid attributes for a `figure.frames[*]` entry?